resources/sass/app.scss

```diff
@@ -242,6 +250,14 @@ legend {
     }
 }
 
+.list-bios {
+    @extend %index-list;
```

Comment:
Yeah I think the multi-class pattern is the better approach generally. To me it's analogous to:

```
.btn {
 // main button styles
}

.btn--success {
  @extend .btn;
  background-color: green;
}
```

I think the extension should happen in the DOM by combining selectors instead of in the Less files.

The other thing is there's nothing "bio" about the `bios-list` right, it's just an `index-list` with a different h3 header style.

It sounds sort of heretical compared to the ways of a few years ago, but I really think "style it in the DOM, extract a component when it reveals itself" is the most maintainable approach nowadays. To me it just comes down to the fact that maintaining markup is usually way easier than maintaining CSS, so the less CSS you write and the less changes you have to make to the CSS, the better, haha...